README.md

```
@@ -90,4 +90,50 @@ git tag -a v1.0.0 -m "Release version 1.0.0"
+
+- 스코프 (선택 사항): 변경된 부분의 범위 (Jira 이슈 키 등)
+
+- 제목: 커밋 내용에 대한 간결한 설명
+
+## 주요 Gitmoji 및 타입
+
+| Gitmoji | 타입 | 설명 |
+|---|---|---|
+| ✨ `:sparkles:` | feat | 새로운 기능을 추가할 때 |
+| 🐛 `:bug:` | fix | 버그를 수정할 때 |
+| 📚 `:books:` | docs | README.md 등 문서를 수정할 때 | <- 이건 깃허브에서 바로 수정 가능하니 무시하셔도 됩니다.
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)